fem/eltrans.hpp

```diff
@@ -53,14 +53,26 @@ class ElementTransformation
 
    ElementTransformation();
 
+   /** @brief Set the integration point @a ip that weights and jacobians will
+       be evaluated at. */
    void SetIntPoint(const IntegrationPoint *ip)
    { IntPoint = ip; EvalState = 0; }
+
+   /// Get a const reference to the currently set integration point.
```

Comment:
This method may benefit from a warning that it can cause trouble if the integration point has not been set yet (dereferencing a null pointer).